templates/bin_setup

```diff
@@ -13,8 +13,12 @@ bundle check || bundle install
 # Set up database and add any development seed data
 bin/rails dev:prime
 
-# Add binstubs to PATH via export PATH=".git/safe/../../bin:$PATH" in ~/.zshenv
-mkdir -p .git/safe
+if [ ! -d .git/safe ] && echo $PATH | grep .git/safe > /dev/null; then
+  echo "-----------------------------------------------------------------------"
+  echo
+  echo "-> When you trust this repo, remember to run: mkdir -p .git/safe"
```

Comment:
The check on the line above makes sure to only show this message to people who have configured their PATH. That way we only show this to those of us who make use of such a feature.